pkg/controller/elasticsearch/initcontainer/prepare_fs.go

```diff
@@ -125,9 +115,6 @@ func NewPrepareFSInitContainer(
 		VolumeMounts: append(
 			PluginVolumes.InitContainerVolumeMounts(),
 			certificatesVolumeMount,
-			scriptsVolume.VolumeMount(),
-			esvolume.DefaultDataVolumeMount,
-			esvolume.DefaultLogsVolumeMount,
 		),
```

Comment:
A comment repeating that some volumes are inherited from the main container may be useful here?